Sources/Sampler/exact_sampler.hpp

```diff
@@ -89,7 +89,7 @@ class ExactSampler : public AbstractSampler {
 
     for (int i = 0; i < dim_; ++i) {
       auto v = hilbert_index_.NumberToState(i);
-      logpsivals_[i] = GetMachine().LogVal(v);
+      logpsivals_[i] = GetMachine().LogValSingle(v);
```

Comment:
Reminder here to discuss wether it makes sense to have a single `LogVal` or add a `LogValSingle` version instead of an if statement in `LogVal`